ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-device-documentation.j2

```diff
@@ -236,6 +236,8 @@
 {% include 'documentation/platform.j2' %}
 {# Router L2 VPN #}
 {% include 'documentation/router-l2-vpn.j2' %}
+{# Router MSDP #}
```

Comment:
please move this up under multicast section and make it start with ##.

Reply:
Done. Thanks!